configure.ac

```diff
@@ -673,6 +677,26 @@ if test "x$included_pcre2" = "xyes"; then
   LIBS="$LIBS $PCRE2_LIBS"
 fi
 
+
+
+# ========
+# MuParser math expression library configuration.
+EXTRA_MUPARSER=
```

Comment:
muParser is available on some systems as an external library (eg Debian), so we'd like to be able to use the system version if it's available ala PCRE2. I can add this functionality later if you prefer.